server/server_test.go

```diff
@@ -15,15 +15,89 @@
 package server
 
 import (
+	"flag"
 	"fmt"
+	"sync"
 	"testing"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+const (
+	postgresDriver   = "postgres"
+	postgresTestUser = "registry_tester"
+	postgresDBConfig = "host=localhost port=5432 user=registry_tester dbname=registry_test sslmode=disable"
```

Comment:
I don't see a way to configure these credentials for testing against a postgres database that has a different test user and configuration; I'd like to add that later. It would be good to also explain how to run the tests against postgres in the main README, possibly in [this section](https://github.com/apigee/registry#optional-running-with-a-postgresql-backend).

Reply:
Agreed. I had a few ideas on how to handle configurability.
1. After addressing #230 this would be like any other configurable value: settable by file, environment, or flag.
2. We can borrow the standard [environment variables](https://www.postgresql.org/docs/9.3/libpq-envars.html) from `psql`.
3. We can add environment variables for this specific testing situation.

How important do you think it is to document this in the README? I think we should wait and see how often developers have a reason to run these locally before investing in ease of setup.